scripting/togglemusic_dhook.sp

```diff
@@ -33,81 +27,89 @@ StringMap g_smCommon;
 StringMap g_smRecent;
 StringMap g_smVolume;
 
-public Plugin myinfo =
+public Plugin myinfo = 
 {
-	name = PLUGIN_NAME,
-	author = "Mitch & Agent Wesker",
-	description = "Allows clients to toggle ambient sounds played by the map",
-	version = PLUGIN_VERSION,
+	name = "Toggle Music", 
```

Comment:
Why was this change needed? You remove the PLUGIN_NAME define but maintain PLUGIN_VERSION.

Reply:
Ime a Freak of Perfectness, the Plugin Name is only used once, the Version 2 Times. This is just optimization work making it look better, there is no actual relevant change.
I even remove all the Spaces between if Arguments and the Brackets in all Plugins i maintain.